README.md

```diff
@@ -7,6 +7,111 @@ The SDK is compatible with:
 * .NET Framework 4.6.1
 * Mono
 
+# Installation
+## Using Nuget Package Manager
+ 1. Right-click on your project in the Solution Explorer and choose **Manage Nuget Packages...**
+ 2. Search for Stormpath. Install the `Okta.Sdk` package.
+
+## Using The Nuget Package Manager Console
+Simply run `install-package Okta.Sdk`. Done!
+
+# Client Configuration
+You can configure the `OktaClient` in one of three ways:
```

Comment:
Before we tell them _how_ to configure the OktaClient, let's explain what it needs: **Okta Org URL and API token**. We can point them to the [docs](https://developer.okta.com/docs/api/getting_started/getting_a_token.html) so it's clear how to generate a token.